Sources/Interaction/Style/InteractorStyleManipulator/index.js

```diff
@@ -420,13 +421,19 @@ function vtkInteractorStyleManipulator(publicAPI, model) {
       model.currentManipulator.getButton() === button
     ) {
       model.currentManipulator.onButtonUp(model._interactor);
-      model.currentManipulator.endInteraction();
-      model.currentManipulator = null;
-      model._interactor.cancelAnimation(publicAPI.onButtonDown);
+      const canEndManipulator = model.currentManipulator.endInteraction();
```

Comment:
that must be documented somewhere the signification of the `endInteraction()` return value.